specification/devcenter/DevCenter/shared/models.tsp

```diff
@@ -6,53 +6,38 @@ import "../main.tsp";
 using Azure.Core;
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace DevCenterService;
 
-@doc("Indicates whether operation status is running, completed, canceled or failed.")
-@lroStatus
-union OperationStatusValue {
-  string,
-
-  @doc("Operation is in progress")
-  Running: "Running",
-
-  @doc("Operation is completed with success")
-  @lroSucceeded
-  Completed: "Completed",
-
-  @doc("Operation was canceled")
-  Canceled: "Canceled",
-
-  @doc("Operation failed")
-  Failed: "Failed",
-}
-
-@doc("The current status of an async operation")
+@doc("The current status of an async operation.")
 @resource("operationstatuses")
 @parentResource(Project)
 model OperationStatus {
   @doc("Fully qualified ID for the operation status.")
-  @key("operationId")
-  @visibility("read")
```

Comment:
Why was this changed to name? We cant change keys.

Reply:
This wasn't the correct key, it should be name. But both should be read-only so that's been fixed. 

"id": "/projects/myProject/operationStatuses/fa067167-e49d-41bd-8dd8-de719b9de3b3",
"name": "fa067167-e49d-41bd-8dd8-de719b9de3b3",